trie-db/CHANGELOG.md

```diff
@@ -6,6 +6,9 @@ The format is based on [Keep a Changelog].
 
 ## [Unreleased]
 
+## [0.22.2] - 2021-01-05
+- Update `hashbrown` to 0.9. [#118](https://github.com/paritytech/trie/pull/118)
```

Comment:
```suggestion
- Update `hashbrown` to 0.9. [#118](https://github.com/paritytech/trie/pull/118)
- Externalize test to internal test crate. [#111](https://github.com/paritytech/trie/pull/111)
```

Reply:
This change is not observable for an external user.